UIKitTeamplate/Payment screen/PayCheckViewController.swift

```diff
@@ -0,0 +1,194 @@
+// PayCheckViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Класс отображающий чек заказа и оплату
+final class PayCheckViewController: UIViewController {
+    // MARK: - Visual Components
+
+    // Кнопка закрытие экрана
+    private let closeButton = UIButton()
+
+    // Картинки узоров
+    private let leftImageView = UIImageView()
+    private let rightImageView = UIImageView()
+    private let centerImageView = UIImageView()
+
+    // Кнопка оплатить
+    private let payButton = UIButton()
+
+    // Лейблы с названием экрана и суммой заказа
+    private let titleLabel = UILabel()
+    private let sumLabel = UILabel()
+
+    // Лейблы с названием продуктов
+    private let americanoLabel = UILabel()
+    private let milkLabel = UILabel()
+    private let espressoLabel = UILabel()
+
+    // Лейблы с ценами продуктов
+    private let firstPriceLabel = UILabel()
+    private let secondPriceLabel = UILabel()
+    private let thirdPriceLabel = UILabel()
+
+    // MARK: - СallBacks
+
+    // Сlosure для показа экрана "Спасибо за заказ"
+    var pushThanks: (() -> ())?
```

Comment:
```suggestion
    var pushThanks: (() -> ())?
```
название не по гайду